api/v1/models/MailingList.js

```diff
@@ -0,0 +1,27 @@
+var Model = require('./Model');
+var User = require('./User');
+var MailingListUser = require('./MailingListUser');
+
+var MailingList = Model.extend({
+	tableName: 'mailing_lists',
+	idAttribute: 'id'
+});
+
```

Comment:
Is there a way to check for collisions in names? seems pretty easy as usage goes on to forget you added a list